requirements/base.in

```diff
@@ -0,0 +1,26 @@
+# Core dependencies for running journals
+
+beautifulsoup4>=4.6.0,<5.0.0        # html parser
+boto==2.49.0                        # python interfaces to Amazon Web Services
+django==1.11.15
+django-filter==1.1.0                # django queryset filtering
+django-extensions==2.0.0            # django custom extensions
+django-model-utils==3.1.1
+django-upload-validator==1.0
+django-storages==1.7.1
+django-waffle==0.13.0
+djangorestframework==3.6.4
+djangorestframework-jwt==1.8.0
+edx-auth-backends==1.1.3
+edx-django-release-util==0.3.1
+edx-django-utils==1.0.1
+edx-drf-extensions==2.0.0
+edx_rest_api_client==1.8.2
+elasticsearch>=5.0.0,<6.0.0
+jsonfield==2.0.2
+mysqlclient==1.3.12
+pytz==2018.7
+requests==2.11.1                    # pinned to resolve dependecy conflict while running pip-compile
+urllib3
+wagtail>=1.13,<1.14
+django-cors-headers==2.3.0
```

Comment:
yeah .in files should have as much as possible unpinned dependencies. I did not unpin them just to avoid breaking anything. However, I think we should first try to unpin them and see if it does cause any issue if it does then we can pin the relevant dependencies.